### ISSUE TITLE
Could you please provide inference code?

### ISSUE THREAD

**Issue by jellyfish1456** [ga642381/Taiwanese-Whisper#2]
Thank you for sharing this awesome work on Taiwanese-Whisper!

Could you please provide the inference code also?

Thank you!

**Comment by ga642381** [comment]
Good Idea! Thank you, @christw16. I will add it when I have the capacity to do that. And PR is also welcome :)

**Comment by pvtoan** [comment]
Hi @ga642381 ,

Thank you for your valuable contribution!

Could you please release your trained model for Taiwanese Whisper?
